bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -558,21 +559,21 @@ void BCStateTran::saveReservedPage(uint32_t reservedPageId,
 
   Assert(!isFetching());
   Assert(reservedPageId < numberOfReservedPages_);
-  Assert(copyLength <= kSizeOfReservedPage);
+  Assert(copyLength <= sizeOfReservedPage_);
 
   psd_->setPendingResPage(reservedPageId, inReservedPage, copyLength);
 }
 
 void BCStateTran::zeroReservedPage(uint32_t reservedPageId) {
-    LOG_INFO(STLogger, "BCStateTran::zeroReservedPage - reservedPageId="
+    LOG_DEBUG(STLogger, "BCStateTran::zeroReservedPage - reservedPageId="

```

Comment:
Was it your intention to commit the demotion of this logging statement from `INFO` to `DEBUG`, and, if so, can I ask why? This change does not seem immediately related to your configurability fix, so I wanted to validate it is intentional and understand the context for it.

Reply:
Thanks for your review! Yes, this is intentional, I was playing around with the number of reserved pages. Logging this for every page in INFO delays start-up time significantly.

Reply:
Thank you for the clarification.